devfile.yaml

```diff
@@ -33,17 +33,51 @@ components:
 
 commands:
 
-  - name: yarn
+  - name: Build
     actions:
       - type: exec
         component: dev
-        command: yarn
+        command: >
+                 yarn
         workdir: /projects/chectl
 
-  - name: yarn test
+  - name: Test
     actions:
       - type: exec
         component: dev
-        command: yarn test
+        command: >
+                 yarn test
         workdir: /projects/chectl
 
+  - name: Run
+    actions:
+      - type: exec
+        component: dev
+        command: >
+                 ./run --help
+        workdir: /projects/chectl/bin
+
+  - name: Install Packager
```

Comment:
not required anymore (only for deprecated binaries)

Reply:
I have removed this command and updated a command to build packaged.
I will update it when it become clean how to use `oclif-dev pack`.

Reply:
Removed